packages/fast-usdc-contract/src/exos/settler.ts

```diff
@@ -362,13 +365,20 @@ export const prepareSettler = (
          *
          * @param {EvmHash} txHash
          * @param {NatValue} fullValue
+         * @param {AccountId | CosmosChainAddress['value']} EUD
```

Comment:
```suggestion
         * @param {AccountIdArg} EUD
```

Reply:
`AccountIdArg` is `AccountId | CosmosChainAddress`, so going to leave as is. Noisier than `string`, but more helpful IMO.